common/setups/rasr/rasr_system.py

```diff
@@ -165,32 +166,45 @@ def _set_scorer_for_corpus(self, eval_corpus_key: str):
         :param eval_corpus_key:
         :return:
         """
-        if self.rasr_init_args.scorer_args is not None:
-            self.glm_files.update(self.rasr_init_args.scorer_args.get("glm_files", {}))
-            self.stm_files.update(self.rasr_init_args.scorer_args.get("stm_files", {}))
+        scorer_args = copy.deepcopy(self.rasr_init_args.scorer_args)
         if self.rasr_init_args.scorer == "kaldi":
             scorer_args = (
-                self.rasr_init_args.scorer_args
-                if self.rasr_init_args.scorer_args is not None
+                scorer_args
+                if scorer_args is not None
                 else dict(mapping={"[SILENCE]": ""})
             )
             self.set_kaldi_scorer(
                 corpus=eval_corpus_key,
                 **scorer_args,
             )
         elif self.rasr_init_args.scorer == "hub5":
-            self.set_hub5_scorer(corpus=eval_corpus_key)
+            scorer_args = (
+                scorer_args if self.rasr_init_args.scorer_args is not None else {}
```

Comment:
```suggestion
                scorer_args if scorer_args is not None else {}
```
Would be more consistent with the change above.